src/koheesio/spark/transformations/drop_column.py

```diff
@@ -45,6 +45,6 @@ class DropColumn(ColumnsTransformation):
     In this example, the `product` column is dropped from the DataFrame `df`.
     """
 
-    def execute(self) -> ColumnsTransformation.Output:
-        self.log.info(f"{self.column=}")
+    def execute(self) -> None:
```

Comment:
execute never returns None - this should be some form of Output

Reply:
Fixed